mod_reforged/hooks/ai/tactical/behaviors/ai_attack_throw_net.nut

```diff
@@ -4,6 +4,18 @@
 	q.onEvaluate = @(__original) function( _entity )
 	{
 		local ret = __original(_entity);
+		// It is a generator in vanilla, but we add the typeof check for maximum
+		// compatibility if another mod changed the original function to not be a generator.
```

Comment:
I don't think we should try to catch bad hooking by other mods and make them compatible.
If someone hooks a generator we should assume that it remains a generator afterwards